sql/flow.go

```diff
@@ -26,21 +23,62 @@ const (
 )
 
 var (
-	DockerClientInit = NewDockerClient
-	ioCopy           = io.Copy
+	DockerClientInit   = NewDockerClient
+	IoCopy             = io.Copy
+	PrintBuildingSteps = printBuildingSteps
+	Println            = fmt.Println
 )
 
 func getContext(filePath string) io.Reader {
 	ctx, _ := archive.TarWithOptions(filePath, &archive.TarOptions{})
 	return ctx
 }
 
+func printBuildingSteps(r io.Reader) error {
+	decoder := json.NewDecoder(r)
+	var prevStream string
+	var currStream string
+	firstMessage := true
+	for {
+		var jsonMessage jsonmessage.JSONMessage
+		if err := decoder.Decode(&jsonMessage); err != nil {
+			if err == io.EOF {
+				break
+			}
+			return err
+		}
+		if err := jsonMessage.Error; err != nil {
+			return err
+		}
+		if jsonMessage.Stream == "\n" {
+			continue
+		}
+		currStream = jsonMessage.Stream
+		// We only print steps which are actually running, e.g.
+		// Step 2/4 : ENV ASTRO_CLI Yes
+		//  ---> Running in 0afb2e0c5ad7
+		if strings.HasPrefix(prevStream, "Step ") && strings.HasPrefix(currStream, " ---> Running in ") {
+			if firstMessage {
+				if _, err := Println("Installing flow.. This might take some time."); err != nil {
+					return err
```

Comment:
```suggestion
				fmt.Println("Installing flow.. This might take some time.")
```
I don't think it is important to check the error for `fmt.Println`, even if `fmt.Println` most likely have an error, then the returned error from here might also not be logged